src/nbl/video/COpenGLCommandPool.h

```diff
@@ -47,6 +64,248 @@ class COpenGLCommandPool final : public IGPUCommandPool
         core::CMemoryPool<core::GeneralpurposeAddressAllocator<uint32_t>,core::default_aligned_allocator,false,uint32_t> mempool;
 };
 
+class NBL_FORCE_EBO COpenGLCommandPool::ICommand
+{
+public:
+    virtual void operator() (IOpenGL_FunctionTable* gl, SOpenGLContextLocalCache* ctxLocal, uint32_t ctxid, const system::logger_opt_ptr logger, const COpenGLCommandBuffer* executingCmdbuf) = 0;
```

Comment:
I think after all the changes, the last argument won't be necessary

Reply:
Yes not necessary now.
Resolved.